### PR TITLE
Fix/13375 photo library access

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
Removing code to ask for photo library access permission as it is not required anymore as per apple's guideline.  

https://developer.apple.com/videos/play/wwdc2020/10652/?time=832

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13375

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/FileScanner/FileScannerCoordinator.swift [hunk 1]
```diff
@@ -183,19 +183,6 @@ class FileScannerCoordinator: NSObject, UIImagePickerControllerDelegate, UINavig
...
-					DispatchQueue.main.async { [weak self] in
-						self?.presentPhotoPicker()
-					}
-				}
-			} else {
-				presentPhotoAccessAlert()
-			}
-			return
-		}
-
```
Comment (30mar) [inline src/xcode/ENA/ENA/Source/Scenes/FileScanner/FileScannerCoordinator.swift:198]: did you also test on iOS12.5 ?

**Review by Ein-Tim — COMMENTED** [review]
This PR does not remove the corrseponding AppStrings / Localizable strings. Why?

File: src/xcode/ENA/ENA/Source/Scenes/FileScanner/FileScannerCoordinator.swift (hunk 1 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/FileScanner/FileScannerCoordinator.swift:198]: @30mar Yes. It is working as expected.

**Review by 30mar — APPROVED** [review]

**Review by KathrinLuecke — APPROVED** [review]
UA approved

**Comment by Ein-Tim** [comment]
Will someone answer my comment ?

**Comment by naveeddotio** [comment]
> Will someone answer my comment [#4655 (review)](https://github.com/corona-warn-app/cwa-app-ios/pull/4655#pullrequestreview-1030121886)?

@Ein-Tim Please check the file changes again.

**Comment by Ein-Tim** [comment]
@naveeddotio I was talking about e.g. this string: https://github.com/corona-warn-app/cwa-app-ios/blob/c009baf4456dd7d03ad0957eac372be392889e70/src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings#L3514

**Comment by naveeddotio** [comment]
> @naveeddotio I was talking about e.g. this string:
> 
> https://github.com/corona-warn-app/cwa-app-ios/blob/c009baf4456dd7d03ad0957eac372be392889e70/src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings#L3514

@Ein-Tim We still need that, there can be a condition where a user explicitly tell the OS not to provide photo access to this app. Please refer to line 41 to 58 in FileScannerCoodinatorViewModel, we want to cover all the cases.

**Comment by Ein-Tim** [comment]
@naveeddotio Got you, thanks for clarifying!

**Review by flxschmidt — APPROVED** [review]

**Pull request merged**